### PR TITLE
Jake/add sentiments

### PR DESCRIPTION
**Pull request by jake-ringdna**
Basic functionality to implement sentiment detection. Uses TextBlob for the heavy lifting. Only surfaces polarity at the moment, but can easily be modified to surface subjectivity, and/or could include confidence in a model that supports it.

Sentiments are stored as timestamps and values, as "measurements" in the call_statistics model, and are processed as "transcript metrics" within the call_metrics_pipeline.

### CODE REVIEW

File: call_metrics/lib/pipelines/model/call_metrics_pipeline.py [hunk 1]
```diff
@@ -34,6 +36,14 @@ def process_transcript_metrics(self, transcript, call_statistics):
         streaks_dict = self.streaks_processor.process(transcript)
         call_statistics.set_transcript_full_streaks(transcript.transcript_id, streaks_dict["full_streaks"])
         call_statistics.set_transcript_active_streaks(transcript.transcript_id, streaks_dict["active_streaks"])
+
+        sentiments = self.sentiments_processor.process(transcript)
+        
+        for timestamp, sentiment_value in sentiments:
+
```
Comment (correlator) [inline call_metrics/lib/pipelines/model/call_metrics_pipeline.py:43]:
```suggestion
```

File: call_metrics/lib/processors/model/sentiments_processor.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+from lib.models.transcript import Transcript
+
+class SentimentsProcessor:
+
+    def process(self, transcript: Transcript):
+        sentiments = []
+
+        for sentence in transcript.sentences:
+
```
Comment (correlator) [inline call_metrics/lib/processors/model/sentiments_processor.py:13]:
```suggestion
```

File: call_metrics/lib/processors/model/sentiments_processor.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+            processed_sentence = TextBlob(sentence.to_string())
+            sentence_sentiment = processed_sentence.sentiment.polarity
+
+            if sentence_sentiment:
+
```
Comment (correlator) [inline call_metrics/lib/processors/model/sentiments_processor.py:18]:
```suggestion
```

File: call_metrics/lib/processors/model/sentiments_processor.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+from lib.models.transcript import Transcript
+
+class SentimentsProcessor:
+
+    def process(self, transcript: Transcript):
+        sentiments = []
+
+        for sentence in transcript.sentences:
+
+            processed_sentence = TextBlob(sentence.to_string())
```
Comment (correlator) [inline call_metrics/lib/processors/model/sentiments_processor.py:14]: Can we remove all of this logic from the sentence model now that it's here?

**Comment by jake-ringdna** [comment]
Bad whitespace removed. Unit tests now utilize in-range values.

File: call_metrics/lib/models/sentence.py [hunk 5]
```diff
@@ -25,9 +24,6 @@ def is_complete(self):
 
     def mark_complete(self):
         self.end_time = self.words[-1].end_time
-        #sentiment = TextBlob(self.to_string()).sentiment
```
Comment (correlator) [inline call_metrics/lib/models/sentence.py:28]: nice.  Also the sentiment and subjectivity instance variables declared on lines 13 and 14

**Pull request merged**